plugins/hydra_ray_launcher/examples/docker/config.yaml

```diff
@@ -0,0 +1,48 @@
+task: 1
```

Comment:
Thank you for adding an example!

Could we only add configs that are actually needed for the example to work?

Some of the configs here are already in teh default config so shouldnt be needed to be put here again

Reply:
generally speaking I've suppressed redundant config fields that won't usually need to be changed. There are some fields that are worth keeping IMO because it allows the user to easily adapt the example to their requirements.